.github/workflows/pkg.pr.new.yml

```
@@ -6,33 +6,44 @@ on:
     types: [submitted]
 
 jobs:
-  deploy-preview:
-    name: Deploy to pkg.pr.new
-    if: github.event.review.state == 'APPROVED' || contains(github.event.pull_request.author_association, 'MEMBER')
+  check-changes:
```

Restore workflow authorization check

```suggestion
  check-changes:
    name: Check for relevant changes to deploy
    if: github.event.review.state == 'APPROVED' || contains(github.event.pull_request.author_association, 'MEMBER')
```